### PR TITLE
Add Cellular 0.0.8

### PR DESCRIPTION
**Pull request by numbermaniac**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/cellular.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version '0.0.8'
+  sha256 '93b9d5a46e2e068f14f5af1c3220294615b607e0dcc1267ca42d58356d812170'
+
+  # github.com/downloads/monofonik was verified as official when first introduced to the cask
+  url "https://github.com/downloads/monofonik/cellular/Cellular-#{version}.dmg"
+  name 'Cellular'
+  homepage 'http://www.flipt.org/#cellular'
+  license :unknown
+
+  app 'Cellular'
```
Comment (reitermarkus) [inline Casks/cellular.rb]: `app` should be changed to `suite` in this case, as this is a folder the application and other files.

Would be great if you could also add a [`zap` stanza](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md). Example [editready.rb](https://github.com/caskroom/homebrew-cask/blob/42437e5fcd93b1a3c95376794e6d5eee2b096160/Casks/editready.rb#L12-L15)

**Comment by Numbermaniac** [comment]
Suite is done, but which files am I supposed to deleting with zap? I can't find them on my computer.

**Comment by reitermarkus** [comment]
@Numbermaniac, usually there are preference files which are created by the application. Try searching for "cellular" inside `~/Library` and see if you can find something.

**Comment by Numbermaniac** [comment]
@reitermarkus Yeah, I searched and nothing showed up, other than a .sfl file for the program but I'm not sure if that's necessary to delete.

**Comment by reitermarkus** [comment]
Please add the `.sfl` file. It's not necessary, but helpful if you want to reset an application or just keep your system clean.

**Comment by Numbermaniac** [comment]
@reitermarkus Done :)

**Comment by reitermarkus** [comment]
Thank you for the contribution. It needed some fixes, so they were made in dd54e245b4eea54d2e50b5135b600d668c2f2b36. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**